README.md

```diff
@@ -47,22 +47,28 @@ Cool! Now you're ready to provision OpenShift clusters
 
 To start, we'll provision the `sample.casl.example.com` cluster defined in the `casl-ansible/inventory` directory. 
 
-**Note**: *It is recommended that you use a different inventory by duplicating the `sample.casl.example.com` directory and keep it elsewhere. In that way, you can update/remove/change your casl-ansble directory without losing your inventory. It may take some time to get the inventory just right, hence it is very beneficial to keep it around for future use without having to redo everything.*
+**Note**: *It is recommended that you use a different inventory by duplicating the `sample.casl.example.com.d` directory and keep it elsewhere. In that way, you can update/remove/change your casl-ansble directory without losing your inventory. It may take some time to get the inventory just right, hence it is very beneficial to keep it around for future use without having to redo everything.*
 
 The following is just an example on how the `sample.casl.example.com` inventory can be used:
 
-1) Edit `casl-ansible/inventory/sample.casl.example.com/inventory/hosts` and edit the `# OpenStack Provisioning Variables` and `# Subscription Management` sections at the top to match your environment/project/tenant. See comments in the file for more detailed information on how to fill these in.
+1) Edit `casl-ansible/inventory/sample.casl.example.com.d/inventory/hosts` and edit the `# OpenStack Provisioning Variables` and `# Subscription Management` sections at the top to match your environment/project/tenant. See comments in the file for more detailed information on how to fill these in.
 
 2) Start the `openstack-client` container.
 ```
-./casl-ansible/docker/openstack-client-centos/run.sh --repository=/path/to/repos/dir/
+docker run -it --name control-host -v $HOME/.ssh:/root/.ssh -v $HOME/.config/openstack:/root/.config/openstack -v $HOME/src:/root/repository -v $HOME/.ansible.cfg:/root/.ansible.cfg docker.io/redhatcop/control-host-openstack bash
 ```
 
+**Note**: If you see an error saying that there are no `config/*.sh` files, try
+temporarily disabling SELinux (`sudo setenforce 0`) and running a new
+container.
+
 3) Run the `end-to-end` provisioning playbook
 ```
-ansible-playbook -i /root/repository/casl-ansible/inventory/sample.casl.example.com.d/inventory /root/repository/casl-ansible/playbooks/openshift/end-to-end.yml
+ansible-playbook -i /root/repository/casl-ansible/inventory/sample.casl.example.com.d/inventory /root/repository/casl-ansible/playbooks/openshift/end-to-end.yml -e openstack_ssh_public_key=openshift
```

Comment:
could we make the added param something like `-e openstack_ssh_public_key=<your_ssh_key_name>`  to indicate that they need to supply whatever name they have for their key? The comment below is good, but would like to make the example a bit more generic as well.